src/components/shared/Entity/EndpointConfig/index.tsx

```diff
@@ -83,12 +84,10 @@ function EndpointConfig({
 
     // Storing if this endpointConfig can be empty or not
     //  If so we know there will never be a "change" to the endpoint config
-    const canBeEmpty = useMemo(() => {
-        return (
-            !connectorTag?.endpoint_spec_schema.properties ||
-            isEmpty(connectorTag.endpoint_spec_schema.properties)
-        );
-    }, [connectorTag?.endpoint_spec_schema]);
+    const canBeEmpty = useMemo(
+        () => configCanBeEmpty(connectorTag?.endpoint_spec_schema),
+        [connectorTag?.endpoint_spec_schema]
+    );
```

Comment:
Since we are using this function in the store want to make sure they use the same logic.